### PR TITLE
bikeshed misspells

### PR DESCRIPTION
**Pull request by krzyzacy**
https://goreportcard.com/report/github.com/kubernetes/test-infra oh we lost 2 percent :-(

/assign @BenTheElder

### CODE REVIEW

File: experiment/commenter/main_test.go [hunk 1]
```diff
@@ -227,7 +227,7 @@ func TestRun(t *testing.T) {
 		{
 			name:    "find none",
 			query:   "none",
-			comment: "this should not happend",
+			comment: "this should not happened",
```
Comment (BenTheElder) [inline experiment/commenter/main_test.go]: s/happened/happen

Comment (krzyzacy) [reply experiment/commenter/main_test.go]: obviously they don't check for grammar errors

**Comment by BenTheElder** [comment]
so actually worse we gained an ineffAssign too 😕 https://goreportcard.com/report/github.com/kubernetes/test-infra#ineffassign

**Comment by BenTheElder** [comment]
Maybe when we get linting by default we can add `ineffassign` to the lint suite.

**Pull request merged**